tests/python/frontend/pytorch/test_forward.py

```diff
@@ -0,0 +1,767 @@
+
+    with torch.no_grad():
+        input_data = torch.rand(input_shape).float()
+        verify_model(Dropout1().float().eval(), input_data=input_data)
+
+def test_forward_slice():
+    torch.set_grad_enabled(False)
+    input_shape = [1, 3, 10, 10]
+
+    class Slice1(Module):
+        def forward(self, *args):
+            return args[0][:, :, :, :3]
+
+    class Slice2(Module):
+        def forward(self, *args):
+            return args[0][0, :, :, :]
+
+    with torch.no_grad():
+        input_data = torch.rand(input_shape).float()
+        verify_model(Slice1().float().eval(), input_data=input_data)
+        verify_model(Slice2().float().eval(), input_data=input_data)
+
+def test_forward_mean():
+    torch.set_grad_enabled(False)
+    input_shape = [1, 3, 10, 10]
+
+    class Mean1(Module):
+        def forward(self, *args):
+            return args[0].mean(2)
+
+    with torch.no_grad():
+        input_data = torch.rand(input_shape).float()
+        verify_model(Mean1().float().eval(), input_data=input_data)
+
+def test_forward_expand():
+    torch.set_grad_enabled(False)
+    input_shape = [1, 3, 10, 10]
+
+    class Expand1(Module):
+        def forward(self, *args):
+            return args[0].expand((3, -1, -1, -1))
+
+    with torch.no_grad():
+        input_data = torch.rand(input_shape).float()
+        verify_model(Expand1().float().eval(), input_data=input_data)
+
+def test_forward_pow():
+    torch.set_grad_enabled(False)
+    input_shape = [1, 3, 10, 10]
+
+    class Pow1(Module):
+        def forward(self, *args):
+            return args[0] ** 2
+
+    with torch.no_grad():
+        input_data = torch.rand(input_shape).float()
+        verify_model(Pow1().float().eval(), input_data=input_data)
+
+def test_forward_chunk():
+    torch.set_grad_enabled(False)
+    input_shape = [1, 3, 14, 14]
+
+    class Chunk1(Module):
+        def forward(self, *args):
+            chunks = args[0].chunk(7, 2)
+            return torch.cat(chunks, 2)
+
+    with torch.no_grad():
+        input_data = torch.rand(input_shape).float()
+        verify_model(Chunk1().float().eval(), input_data=input_data)
+
+# Model tests
+def test_resnet18():
+    torch.set_grad_enabled(False)
+    verify_model("resnet18")
+
+def test_squeezenet1_0():
+    torch.set_grad_enabled(False)
+    verify_model("squeezenet1_0")
+
+def test_squeezenet1_1():
+    torch.set_grad_enabled(False)
+    verify_model("squeezenet1_1")
+
+def test_mobilenet_v2():
+    torch.set_grad_enabled(False)
+    verify_model("mobilenet_v2")
+
+def test_densenet121():
+    torch.set_grad_enabled(False)
+    verify_model("densenet121")
+
+def test_inception_v3():
+    torch.set_grad_enabled(False)
+    verify_model("inception_v3")
+
+def test_googlenet():
+    torch.set_grad_enabled(False)
+    verify_model("googlenet")
+
+def test_mnasnet0_5():
+    torch.set_grad_enabled(False)
+    verify_model("mnasnet0_5")
+
+#TODO: Fix VGG and AlexNet issues (probably due to pooling)
```

Comment:
What's the issue with pooling?

Reply:
Not sure if pooling (only an assumption) but VGG and AlexNet have bad accuracy on GPU. Working on it separately.